source/_posts/jonber.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 딥백수 그리고 존버
+permalink : Jonber
+date: 2020-08-16 16:06:38
+tags: 존버, jonber, 딥백수, 회고
```

Comment:
```suggestion
tags: 
- 존버
- jonber
- 딥백수
- 회고
```

Reply:
fyi, this is YAML syntax.  https://symfony.com/doc/current/components/yaml/yaml_format.html#collections